news/2024/2024-01-15-osutaiko-osucatch-osumania-4k-world-cup-staff-applications.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: osu!taiko and osu!catch World Cup 2024 Staff Applications
+date: 2024-01-15 12:00:00 +0000
+---
+
+As the 2024 osu! World Cup kicks off with the osu!mania 7K World Cup, preparations for the next World Cups are already underway. Read on to find out how you can contribute!
+
+![](/wiki/shared/news/banners/OWC_Master_Logo.png)
+
+As with every year, we rely heavily on community members for help with many aspects of tournament production. This time, we're mainly looking for mappers and playtesters for the modes listed in the next section.
+
+Mappers will be tasked with creating beatmaps (many times of tailor made, yet-to-be-released [osu! originals](/wiki/osu!_originals)) based on a specification given by each mode's mappooling team. Mappers will be expected to incorporate feedback from the mappoolers and/or playtesters in an iterative process to refine the map until it is in a shippable state.
+
+Playtesters will be responsible for playing and giving feedback for a plethora of beatmaps (as requested by the mappooling team) to gauge aspects such as difficulty, suitability for a certain slot, skill representativity among others (specific workflows vary for each team). Playtesters are generally expected to have considerable skill on their respective game mode.
+
+Please note that if you are accepted into either team, you will not be eligible to participate as a player in that tournament.
+
+## osu!taiko World Cup 2024
+
+The osu!taiko World Cup mappooling team headed by [Nurend](https://osu.ppy.sh/users/9905079) and [Raphalge](https://osu.ppy.sh/users/3918650) is looking for **mappers** and **playtesters**. To sign up, follow [this link](https://forms.gle/Z3nR1W8tcY4rCakf8). Player registrations for TWC 2024 are slated to open Feb 15. 
+
+## osu!catch World Cup 2024
+
+The osu!catch World Cup mappooling team headed by [wwwww](https://osu.ppy.sh/users/8434466) and [Yoshi_green](https://osu.ppy.sh/users/1035891) is looking for **mappers**, **map modders**, and **playtesters**. To sign up, follow [this link](https://forms.gle/p3BVaqXx77S2DoET6). Player registrations for CWC 2024 are slated to open Apr 18.
+
+---
+
+Think you got what it takes to staff for some of the most prestigious tournament series of the game? Then don't waste a second, and submit a form to your preferred World Cup. The osu! World Cups team will be forever grateful for your help with furthering the tournament scene's growth! 
```

Comment:
grammar pedantry

```suggestion
Think you got what it takes to staff for some of the most prestigious tournament series of the game? Then don't waste a second, and submit a form to your preferred World Cup. The osu! World Cup team will forever be grateful for your help with furthering the tournament scene's growth! 
```

Reply:
I think osu! World Cups team is the correct usage here since it refers to the collective "World Cups" and not just the osu! World Cup team (which is a thing by the way)

Reply:
then

```suggestion
Think you got what it takes to staff for some of the most prestigious tournament series of the game? Then don't waste a second, and submit a form to your preferred World Cup. The World Cup team will forever be grateful for your help with furthering the tournament scene's growth! 
```

Reply:
"osu! World Cups team" has never been used in this repository

Reply:
Terminology for this sort of thing has been a mess, historically. It would be good to standardize it somewhat I think, hence the usage. I'm fine with World cup team as well, let's go with that.

Reply:
resolved in [585696f](https://github.com/ppy/osu-wiki/pull/10791/commits/585696f0d4e0195fcf7ee1f8de2b48330b4d4d56)